.github/ISSUE_TEMPLATE/2_bug_report_linux.yml

```diff
@@ -98,6 +98,9 @@ body:
 
 
           Without the log file, this issue will be closed for ignoring the issue template.
+
+          Disclaimer: The log file may contain playback history or other identifiable information, like hardware used.
```

Comment:
if you mean for this to be in a new paragraph you need to add two newlines, otherwise it's just a normal newline and will be directly underneath the previous sentences without any space between them.

Reply:
What do you think is better? I add line break to make it not long in source file. I would think it is ok to have one line in rendered preview.

Reply:
sry we are probably talking about different line breaks here. i meant the one between "Without the log file, this issue will be closed for ignoring the issue template." and "Disclaimer: ...".

currently it will look like this:
<img width="869" alt="Screenshot 2024-07-19 at 00 02 44" src="https://github.com/user-attachments/assets/7f295fa4-dbdc-44d0-ab65-1de959a20c33">

and i thought you might want this instead (one additional newline needed like on the other paragraphes):
<img width="871" alt="Screenshot 2024-07-19 at 00 03 17" src="https://github.com/user-attachments/assets/a5b242d1-a8fa-48e8-abc7-e7369cef185f">

Reply:
I personally think separated by paragraphs looks better. I assume that was the intention but no biggie if you prefer the other way around.